docs/dev/build_raspbian.md

```diff
@@ -22,7 +18,6 @@ Native compilation of the OpenVINO Runtime is the most straightforward solution.
 2. Clone the repositories:
```

Comment:
repositories -> repository

Reply:
Applied the change with https://github.com/openvinotoolkit/openvino/pull/18764/commits/33c23b06303c3a950833a8fc9a752295a7e3f7f0